Source/ShapeMenu/Views/RezizableView/ResizableView.swift

```diff
@@ -63,9 +71,18 @@ class ResizableView: UIView {
         self.updateDragHandles()
     }
     
+    override func removeFromSuperview() {
+           topLeft.removeFromSuperview()
```

Comment:
maybe find a way to iterate thru these views (add them to an array?) to not remove then one by one here from the superview

Reply:
Done